apps/site/lib/site_web/controllers/schedule/timetable_controller.ex

```diff
@@ -58,16 +58,63 @@ defmodule SiteWeb.ScheduleController.TimetableController do
       all_stops: all_stops
     } = build_timetable(conn.assigns.all_stops, timetable_schedules)
 
+    canonical_rps =
+      route.id
+      |> RoutePatterns.Repo.by_route_id(direction_id: direction_id, canonical: true)
```

Comment:
nit-pick: I thought the linter usually complains about piping into a single function

Reply:
Looks like that dotcom hasn't opted into that particular check (`Credo.Check.Readability.SinglePipe`  configured with `false` [here](https://github.com/mbta/dotcom/blob/5f720af3324ac8b60aaca29f5f077195270f99d6/config/.credo.exs#L150). But I'm a fan of that check & will update here!

Reply:
Okay, probably getting confused with Pick Assist configuration